5-appinfra/modules/cicd-pipeline/variables.tf

```diff
@@ -22,6 +22,12 @@ variable "region" {
   description = "CI/CD Region (e.g. us-central1)"
 }
 
+variable "cluster_services_accounts" {
```

Comment:
tought: this can be named a more generic variable, like additional_service_accounts or addicionail_artifact_registry_members